### ISSUE TITLE
haml Gem not compatible with Ruby 3.2 anymore

### ISSUE THREAD

**Issue by thomaswitt** [antifuchs/gmail-britta#34]
~/.rbenv/versions/3.2.0/lib/ruby/gems/3.2.0/gems/haml-3.1.8/lib/haml/version.rb:78:in `revision_number': undefined method `exists?' for File:Class (NoMethodError)

haml is outdated, should be updated.

**Comment by DanceMore** [comment]
I don't think this is maintained anymore. no commits since Oct 2018.

sad, this was a super useful gem in one of my favorite utilities I ever wrote. I used to automatically sort work email based on ActiveDirectory OU; and I also had a huge list of Vendor Domains by category that would also auto-sort.

`gmail-britta` was the important component really doing the work, I just wrote some loops around it.

I was daydreaming about re-creating a modern version for my personal life but might need to find alternatives. a quick search didn't actually turn up any high-quality looking libraries for Gmail XML filters :(

**Comment by antifuchs** [comment]
Yeah, I'll be honest, it's been very tempting to mark this repo as deprecated, archive it and move on. If somebody wants to take over maintainership I'd be more than happy to hand it off!

**Comment by thomaswitt** [comment]
@antifuchs I don't have immediate time for it but in a few weeks I might be able to do some bugfixes. So if you want to transfer, happy to
